piperider_cli/profiler/profiler.py

```diff
@@ -502,7 +502,9 @@ def _profile_histogram(
             if i != num_buckets - 1:
                 cases += [(column < bound, i)]
             else:
-                cases += [(column <= bound, i)]
+                cases += [(column < bound, i)]
+                # equal comparing
+                cases += [(func.abs(column - bound) < interval / 100, i)]
```

Comment:
Would it better to change to `[(column < bound + interval /100, i)]`?

Reply:
OK, it seems to be easier and did not involve database function `abs`